src/main/java/Dtos/PurchaseCartDetailsDto.java

```diff
@@ -16,4 +16,24 @@ public PurchaseCartDetailsDto(List<Integer> basketsToBuy, String cardNumber, Str
     public List<Integer> getBasketsToBuy() {
         return basketsToBuy;
     }
+
```

Comment:
this is good. think about passing all the basket and not the integer, so when you choose you pass the integer - the basket id to the service